test/test_utils/ecs.py

```diff
@@ -764,7 +765,7 @@ def ecs_training_test_executor(cluster_name, cluster_arn, training_command, imag
 
 
 def setup_ecs_inference_service(
-        docker_image_uri, framework, cluster_arn, model_name, worker_instance_id, num_gpus=None, region=DEFAULT_REGION
+        docker_image_uri, framework, cluster_arn, model_name, worker_instance_id, num_gpus=None, region=DEFAULT_REGION, ACCELERATOR_TYPE=None
```

Comment:
Change `ACCELERATOR_TYPE` to `accelerator_type`.

An all-caps variable name indicates that the variable is a Global Variable.